README.md

```diff
@@ -1 +1,4 @@
-# mins
+# *Minesweeper*
```

Comment:
You should change italic style. Remove it.

Reply:
Done. Replaced with bold